src/Modal/ModalContents.js

```diff
@@ -286,6 +286,7 @@ ModalContents.defaultProps = {
   footerClass: 'modal-footer',
   headerClass: 'modal-header',
   modalClass: 'modal modal-large',
+  modalWrapperClass: 'modal-wrapper',
```

Comment:
Just a heads up, we do make use of `modalWrapperClass` in `dcos-ui`. So maybe in this case it would make sense to add `modal-wrapper` as an inline string when creating the classes for this element?